src/python/foglamp/core/server.py

```diff
@@ -184,11 +186,35 @@ def start(cls):
 
         # see http://host:<core_mgt_port>/foglamp/service for registered services
 
-    @staticmethod
-    def stop_storage():
-        # TODO: make client call to service mgt API to ask to shutdown storage?
+    @classmethod
```

Comment:
This work is not in the scope of this ticket. Please remove it.

Reply:
Please refer TODO.  Even ping (_check service ability is also just for quick there only) else ping will also be done by microservice mgt API.

Reply:
I see because it has been removed from storage. Personally I'm fine with breaking shutdown on storage as we originally had a temporary fix. We can finish off the client code here. Create a separate ticket for shutdown so that there is separation of concerns on this review. Then we can address shutdown properly when the time comes.

Reply:
refer FOGL-653 and a PR (on top of this PR itself, which allows we are not breaking anything and allows to fix FOGL-653 post storage engine migration)

Reply:
Also I believe 
1. this is a matter of clean-up and  not to break existing code as its tightly coupled in storage client and we used that to stop storage vs SoC (i.e. Fix is itself in its  concern area)
2. a step towards FOGL-653 hence TODO